ott/core/potentials.py

```diff
@@ -0,0 +1,161 @@
+from typing import Any, Callable, Dict, Sequence, Tuple
+
+import jax
+import jax.numpy as jnp
+import jax.scipy as jsp
+import jax.tree_util as jtu
+from typing_extensions import Literal
+
+from ott.geometry import pointcloud
+
+__all__ = ["DualPotentials", "EntropicPotentials"]
+Potential_t = Callable[[jnp.ndarray], float]
+
+
+@jtu.register_pytree_node_class
+class DualPotentials:
+  r"""The Kantorovich dual potentials :math:`f` and :math:`g`.
+
+  :math:`\nabla g` transports the source distribution to the target distribution
```

Comment:
This is only valid for the W_2 case, so maybe this comment should be removed. Here we should just say that `f` and `g` are a pair of functions, candidates for the dual OT Kantorovich problem, supposedly optimal for a given pair of measures.

Reply:
Thanks, done!